Corona-Warn-App/src/main/res/layout/fragment_information.xml

```diff
@@ -117,11 +112,24 @@
                     android:layout_height="wrap_content"
                     android:layout_marginTop="@dimen/spacing_small"
                     android:focusable="true"
-                    android:text="@{FormatterInformationHelper.formatVersion()}"
+                    tools:text="v1.8.0-RC1"
```

Comment:
hardcoded? 
If this is intended to be this way, leave it there..
Just not sure about this..

Reply:
`tools:text` is just displayed in the layout preview in Android Studio.